readme.md

```diff
@@ -42,11 +42,11 @@ readPkg(path.join('unicorn', 'package.json')).then(pkg => {
 
 ## API
 
-### readPkg([path], [options])
+### readPkg([path], [normalize])
 
 Returns a `Promise` for the parsed JSON.
 
-### readPkg.sync([path], [options])
+### readPkg.sync([path], [normalize])
```

Comment:
This change is not correct.

Reply:
I restored this due to https://github.com/sindresorhus/read-pkg/pull/7/commits/f1051a677571b1bd47cd933ef67b56e0ed960023